session/SessionHandler.php

```diff
@@ -271,5 +271,5 @@ public function validateId($session_id) {}
      * </p>
      * @return bool
      */
-    public function updateTimestamp($session_id, $session_data) {}
+    public function updateTimestamp(string $id, string $data): bool {}
```

Comment:
We can remove this method as well as `validateId()`: they are neither in https://www.php.net/manual/en/class.sessionhandler.php nor in https://github.com/php/php-src/blob/master/ext/session/session.stub.php#L102.
These functions are already in SessionUpdateTimestampHandlerInterface in the same file:
![1](https://user-images.githubusercontent.com/33625946/119645103-f34a6480-be1d-11eb-9be9-dbee2a9066db.png)

Checked where did they come from: https://github.com/JetBrains/phpstorm-stubs/pull/294/files/95d84d28035bdb56a30e5aeebfb6fa068c4e8c32#diff-17c933311fd7e06dd8c1fe0aef6896ddb65d877b9d50eaef405f61f5707269faR262 - they were added by a user in 2017. By the looks of it, it's safe to remove them now.

Reply:
Done